src/app/components/media/Similarity/MediaItem.js

```diff
@@ -307,8 +312,8 @@ const MediaItem = ({
         </Box> : null }
       { canDelete && !canSwitch ?
         <Box>
-          <IconButton onClick={openDialog}>
-            <RemoveCircleOutlineIcon />
+          <IconButton onClick={handleDelete}>
```

Comment:
@danielevalverde please be sure you merged the code correctly (and/or that your branch is up-to-date with `develop`) - on `develop` this is now `openDialog`